CONTRIBUTING.rst

```diff
@@ -0,0 +1,197 @@
+In the top tab you will click on "Help" and then click on "About". In the pop window you will see information about the version of Jupyter that you are running. You will see "The version of the notebook server is:".
+If you are working in development mode, you will see that your version of Jupyter notebook will include the word "dev". If it does not include the word "dev", you are currently not working in development mode and should follow the steps below to uninstall and reinstall Jupyter.
+
+Troubleshooting the Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not see that your Jupyter NbClassic is not running on dev mode, it's possible that you are
+running other instances of Jupyter NbClassic. You can try the following steps:
+
+1. Uninstall all instances of the notebook package. These include any installations you made using
+   pip or conda.
+2. Run ``python3 -m pip install -e .`` in the notebook repository to install the notebook from there.
+3. Run ``npm run build`` to make sure the Javascript and CSS are updated and compiled.
+4. Launch with ``python3 -m nbclassic --port 8989``, and check that the browser is pointing to ``localhost:8989``
+   (rather than the default 8888). You don't necessarily have to launch with port 8989, as long as you use
+   a port that is neither the default nor in use, then it should be fine.
+5. Verify the installation with the steps in the previous section.
+
+
+Rebuilding JavaScript and CSS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is a build step for the JavaScript and CSS in the nbclassic.
+To make sure that you are working with up-to-date code, you will need to run
+this command whenever there are changes to JavaScript or LESS sources::
+
+    npm run build
+
+**IMPORTANT:** Don't forget to run ``npm run build`` after switching branches.
+When switching between branches of different versions (e.g. ``4.x`` and
+``master``), run ``pip install -e .``. If you have tried the above and still
+find that the notebook is not reflecting the current source code, try cleaning
+the repo with ``git clean -xfd`` and reinstalling with ``pip install -e .``.
+
+Development Tip
+"""""""""""""""
+
+When doing development, you can use this command to automatically rebuild
+JavaScript and LESS sources as they are modified::
+
+    npm run build:watch
+
+Git Hooks
+"""""""""
+
+If you want to automatically update dependencies and recompile JavaScript and
+CSS after checking out a new commit, you can install post-checkout and
+post-merge hooks which will do it for you::
+
+    git-hooks/install-hooks.sh
+
+See ``git-hooks/README.md`` for more details.
+
+
+Running Tests
+-------------
+
+Python Tests
+^^^^^^^^^^^^
+
+Install dependencies::
+
+    pip install -e '.[test]'
+
+To run the Python tests, use::
+
+    pytest
+
+If you want coverage statistics as well, you can run::
+
+    py.test --cov nbclassic -v --pyargs notebook
```

Comment:
> py.test --cov nbclassic -v --pyargs notebook

Is this command correct?

Reply:
No, it should be `py.test --cov nbclassic -v --pyargs nbclassic`. I have pushed the needed change.